data/human/free worlds epilogue.txt

```diff
@@ -197,3 +197,92 @@ mission "FW Epilogue: Danforth"
+				decline
+		event "plasma repeater advanced" 95
+		event "plasma repeater basics" 155
+	npc disable save
+		government "Test Dummy"
+		personality disables heroic staying
+		ship "Leviathan (Plasma Repeater)" `Hunk`
+	on complete
+		log `Assisted Barmy Edward with another weapon test: this time, an impressive upgrade to the plasma cannon. Increased fire rate, even at lower damage per hit, comes out with better overall results.`
+		conversation
+			scene "scene/plasma_scene"
+			`You head to the hangar to meet up with Barmy Edward, and find him tinkering with a large gun. It resembles an old Plasma Cannon, but it is somewhat different. Presumably, this is the weapon that you were just attacked with in orbit. "It runs much hotter, you see," he explains. "But the results are impressive."`
+			choice
+				`	"What was that?"`
+					goto what
+				`	"It looks impressive, but it didn't do too much damage."`
+					goto damage
+			label what
+			`	"Heh, it's been a while since we've seen a good upgrade to the cannon." he says. "That was the Plasma Repeater. Of course, it wasn't easy to make a better iteration of something that countless fools have failed to improve on."`
+			`	"When will it be available for sale?" you ask.`
+				goto available
+			label damage
+			`	"Well, you are famous for a reason I suppose." he says. "The idea is that by shrinking the concentrations of superheated plasma, we can offload many more bursts onto a target. The effects on said target result in faster buildup of heat and, of course, damage. Of course, big ships can ward off these effects much longer than a smaller ship can. It's always been a notable issue with the original design, but it's a hell of a lot more impressive."`
+			`	"That is an interesting idea," you reply. "Are they available for sale yet?"`
+			label available
+			`	"Well, I should have an initial model for sale in a month or so," he says. "I'll be interested to see what uses you or anyone else might find for them. Anyway, thanks for your help."`
+			`	"How about we celebrate with some drinks? I know a quaint little bar not far from here."
+			choice
+				`	"I have time; lets celebrate!"`
+				`	"I'll pass on that, but it was good checking in on you."`
+					decline
+			`	You follow Barmy into the bar. Even with the smear of grease still on his face, he doesn't stick out from the crowd. You both stand at the bar and order your drinks. Barmy's order catches you off guard, and he notices your expression. "What? A man who works with his hands can't enjoy a drink with some age?" You chuckle, and you both tell stories of your early lives. He tells how he had some exciting adventures in his youth among the stars but found his passion tinkering and finding ways to improve on outdated equipment.`
```

Comment:
same advice for the next few lines as the advice for this section in the on offer